src/MCPServer/install/README.md

```diff
@@ -153,11 +155,18 @@ This is the full list of variables supported by MCPServer:
     - **Type:** `int`
     - **Default:** `""`
 
-- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCHDIRECTORIESPOLLINTERVAL`**:
-    - **Description:** time in seconds between filesystem poll intervals.
-    - **Config file example:** `MCPServer.watchDirectoriesPollInterval`
```

Comment:
I've changed this for consistency, but it may be worth reverting. As far as I can tell it's not set by any ansible scripts, but I may have missed some.